draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1626,12 +1633,17 @@ client uses `get-all-by-hash`, then it will already have the new STH.
 
 ### Evaluating compliance
 
-To be considered compliant, a certificate MUST be accompanied by at least one
-valid SCT. A certificate not accompanied by any valid SCTs MUST NOT be
-considered compliant by TLS clients.
+To be considered compliant with this specification, a certificate MUST be
```

Comment:
The idea of "at least one valid SCT" was to specify the lowest common denominator level of compliance upon which all local client policies would build.